travis/test_server.py

```diff
@@ -191,7 +192,14 @@ def setup_server(db, odoo_unittest, tested_addons, server_path,
     if preinstall_modules is None:
         preinstall_modules = ['base']
     print("\nCreating instance:")
-    subprocess.check_call(["createdb", db])
+    start_dbtmpl = True
+    try:
+        subprocess.check_call(["createdb", db])
+    except subprocess.CalledProcessError:
+        start_dbtmpl = False
+        print("Using previous openerp_template database.")
+    if not start_dbtmpl:
```

Comment:
Python has a `try ... except ... else ...` compound command: https://docs.python.org/2/reference/compound_stmts.html#try
If no exception is fired in the try block, execution will continue in the else block.
So, you don't need `start_dbtmpl`, and just replace this if with the `else:` part of the try command.